DOCS-5782-Visualized-quick-filtering (#42)
mulesoft/docs-visualizer: modules/ROOT/pages/visualizer-app-network.adoc

Use Anypoint Visualizer to view a graph of your application network.

> The topology of the application network is automatically generated by Anypoint Visualizer.

Reviewer: Can you make this active? "Anypoing Visualizer automatically generates the topology of the application."

Author: ```suggestion
Anypoint Visualizer automatically generates the topology of the application.
```

> To maximize visualization readability, consumer nodes (nodes that only have outbound connections), are shown at the top, and all backend systems

Reviewer: Remove comma after  (nodes that have only outbound connections).

Reviewer: I would probably split this long sentence into two sentences.

Author: ```suggestion
To maximize visualization readability, consumer nodes (nodes that only have outbound connections) are shown at the top. All backend systems are also shown.
```

> (nodes with inbound connections only) are shown at the bottom.

Reviewer 2: We'll go with the version from client apps here!

Author: This will be reflected when I resolve the merge conflict.

> When the `ClientId` is detected for a consumer node, it is grouped by `ClientId` -- otherwise, it is grouped under the *External Traffic* node.

Reviewer: I am not sure about the dash here (looks like two hyphens). Maybe check the Style Guide for rules on using dashes.

Reviewer: What is grouped by 'Client Id"? It sounds like just one consumer node but I think you might all nodes associated with the 'ClientID' are grouped.

Author: ```suggestion
When the `ClientId` is detected for a consumer node, the nodes are grouped by `ClientId`. Otherwise, they are grouped under the *External Traffic* node.
```

== View Information About a Node in Your Network

When you select a node within the graph, the node and its dependency stack are highlighted and all unconnected nodes are unavailable.
Anypoint Visualizer displays information about the node in a card. This card is divided into the following tabs:

The *Details* tab contains:

* *Details*: Provides information about the node.
* *Customization*: Provides information about the layer to which the node is assigned and the tags assigned to the node. You can edit the node's display name or change the layer and tags from this section.

The *Monitoring* tab contains:

* *Metrics summary*: Provides monitoring metrics for the selected node, including Errors count, avg. response time, avg. throughput, avg. CPU consumption, and avg. memory utilization.
* *Metrics chart*: Shows the time-series chart of the selected metric in the *Metric* card.

When you select a node, you can click *Select dependents and dependencies*, which selects all nodes related to the service you selected.
If a single node is selected and does not have any dependents or dependencies, this option is hidden.

> If you select multiple nodes, you can also click *Hide selected services* to hide the selected nodes, or you can click *Hide other services* to hide services that are unrelated to the selected nodes.

Reviewer 2: I'm playing around with a different order. Let me know what you think! 

You can hide nodes through what you select in the canvas.
- To hide a single node, select it and click Hide selected service.
- To hide multiple nodes, multiselect them and click Hide selected services.
- To hide all of the services other than the selected nodes, you can click Hide other services. If you are hoping to hide everything that is not related to a given dependency stack, select a node and click *Select dependents and dependencies*, which selects all nodes related to the service you selected. Then click Hide other services.

Author: ```suggestion
You can hide nodes through what you select in the canvas.

* To hide a single node, select it, and then click *Hide selected service*.
* To hide multiple nodes, multiselect them, and then click *Hide selected services*.
* To hide all of the services other than the selected nodes, you can click *Hide other services*. If you want to hide everything that is not related to a given dependency stack, select a node and then click *Select dependents and dependencies*, which selects all nodes related to the service which you selected. Then, click *Hide other services*.
```

== View Information About a Node in Your Network

When you select a node within the graph, the node and its dependency stack are highlighted and all unconnected nodes are unavailable.
Anypoint Visualizer displays information about the node in a card. This card is divided into the following tabs:

The *Details* tab contains:

* *Details*: Provides information about the node.
* *Customization*: Provides information about the layer to which the node is assigned and the tags assigned to the node. You can edit the node's display name or change the layer and tags from this section.

The *Monitoring* tab contains:

* *Metrics summary*: Provides monitoring metrics for the selected node, including Errors count, avg. response time, avg. throughput, avg. CPU consumption, and avg. memory utilization.
* *Metrics chart*: Shows the time-series chart of the selected metric in the *Metric* card.

When you select a node, you can click *Select dependents and dependencies*, which selects all nodes related to the service you selected.

> If a single node is selected and does not have any dependents or dependencies, this option is hidden.

Author: ```suggestion
If a single node is selected and does not have any dependents or dependencies, this option is hidden.

```

== View Information About a Node in Your Network

When you select a node within the graph, the node and its dependency stack are highlighted and all unconnected nodes are unavailable.
Anypoint Visualizer displays information about the node in a card. This card is divided into the following tabs:

> The *Details* tab contains:

Author: ```suggestion
The *Details* tab contains:

```

> * *Metrics summary*: Provides monitoring metrics for the selected node, including Errors count, avg. response time, avg. throughput, avg. CPU consumption, and avg. memory utilization.

Author: ```suggestion

* *Metrics summary*: Provides monitoring metrics for the selected node, including Errors count, avg. response time, avg. throughput, avg. CPU consumption, and avg. memory utilization.
```

* *Metrics chart*: Shows the time-series chart of the selected metric in the *Metric* card.

> When you select a node, you can *Select dependents and dependencies*, which selects all nodes related to the service you selected.

Reviewer: I think you need a verb before *Select dependents and dependencies". Not sure it should be click or select.

Author: ```suggestion
When you select a node, you can click *Select dependents and dependencies*, which selects all nodes related to the service you selected.
```

If a single node is selected and does not have any dependents or dependencies, this option is hidden.

> If you select multiple nodes, you can also click *Hide selected services* to hide the nodes which you have selected, or you can click *Hide other services* to hide services that are unrelated to the selected node or nodes.

Reviewer: "that you have selected,"

Reviewer: This wording is a bit shorter:

If you select multiple nodes, you can click *Hide selected services* to hide the selected nodes, or *Hide other services* to hide services that are unrelated to the selected nodes.

In this context, "nodes" works for one node or multiple nodes.

Author: ```suggestion
If you select multiple nodes, you can also click *Hide selected services* to hide the selected nodes, or you can click *Hide other services* to hide services that are unrelated to the selected nodes.
```
